DigitalEcosystem/exploratory/initial_work/initial_bg_work_commentated/README.md

```diff
@@ -0,0 +1,3 @@
+# Initial Bandgap Work
+
+This notebook contains some initial work that was performed on the problem of predicting the DFT bandgap of 2D materials.
```

Comment:
We need a README at the top level explaining the content of the repository and the directory structure adopted

Reply:
Added the folder structure to the readme